workflows/mag.nf

```diff
@@ -590,7 +591,17 @@ workflow MAG {
             ch_versions = ch_versions.mix(SPADESHYBRID.out.versions.first())
         }
     } else {
-        ch_assemblies = ch_input_assemblies
+        ch_assemblies_split = ch_input_assemblies
+            .branch { meta, assembly ->
+                gzipped: assembly[0].getExtension() == "gz"
```

Comment:
What is the purpose of the `[0]` Shouldn't there only be one assembly fasta file per sample??

Reply:
getExtension() didn't work without it - I think the structure of the channel is actually `[meta, [assembly]]`, so we need to escape the list environment to get to the file.

Reply:
Ah makes sense, thanks! othersotherwise GTG!